### PR TITLE
Db creation

### PR DESCRIPTION
**Pull request by jgonvers**
created the migration and association for the database. Added default value to user.role (user), event.status (pending), teammate_link (pending).

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :events, through: :attendance
+  has_many :reviews
+  has_many :teammates
```
Comment (maxjost) [inline app/models/user.rb]: through: :TeammateLink

**Comment by maxjost** [comment]
I've added one comment

File: app/models/user.rb (hunk 1 above)
Comment (jgonvers) [reply app/models/user.rb]: done

**Review by maxjost — COMMENTED** [review]
good

**Pull request merged**